### PR TITLE
Add `National Dutch Championship` series to tournaments

### PR DESCRIPTION
**Pull request by MrHeliX**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/NDC/2022/en.md [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=NL }:: [Skyrovania](https://osu.ppy.sh/users/4696315) |
+
+## Mappools
+
+### Grand Finals
+
+**[Download the mappack here!](https://tourney.huismetbenen.nl/10/mappools/gf)**
+
+- Nomod
+  1. [Fuki Commune - Kurui Zake Setsugekka (-Kirigiri) [Destiny]](https://osu.ppy.sh/b/2995484)
```
Comment (iSlodinxOsu) [inline wiki/Tournaments/NDC/2022/en.md:124]:
```suggestion
  1. [Fuki Commune - Kurui Zake Setsugekka (-Kirigiri) [Destiny]](https://osu.ppy.sh/beatmapsets/1457529#osu/2995484)
```

File: wiki/Tournaments/NDC/2022/en.md [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+  2. [pm04034 - Sugary Mocaccino (Astronic) [Caffinated Adrenaline]](https://osu.ppy.sh/b/3251854)
```
Comment (iSlodinxOsu) [inline wiki/Tournaments/NDC/2022/en.md:125]:
```suggestion
  2. [pm04034 - Sugary Mocaccino (Astronic) [Caffinated Adrenaline]](https://osu.ppy.sh/beatmapsets/1592198#osu/3251854)
```

File: wiki/Tournaments/NDC/2022/en.md [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+  3. [Yousei Teikoku - Autoscopy (moph) [Extreme]](https://osu.ppy.sh/b/3333699)
```
Comment (iSlodinxOsu) [inline wiki/Tournaments/NDC/2022/en.md:126]:
```suggestion
  3. [Yousei Teikoku - Autoscopy (moph) [Extreme]](https://osu.ppy.sh/beatmapsets/1633220#osu/3333699)
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/NDC/2022/en.md [hunk 4]
```diff
@@ -0,0 +1,504 @@
...
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=NL }:: [Lilily](https://osu.ppy.sh/users/6502403) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=NL }:: [Skyrovania](https://osu.ppy.sh/users/4696315) |
+
+## Mappools
+
+### Grand Finals
+
+**[Download the mappack here!](https://tourney.huismetbenen.nl/10/mappools/gf)**
+
+- Nomod
```
Comment (0x84f) [inline wiki/Tournaments/NDC/2022/en.md:123]: This needs to be `No Mod` for all mentions across all 3 wiki pages

File: wiki/Tournaments/NDC/2022/en.md [hunk 5]
```diff
@@ -0,0 +1,504 @@
...
+  4. [MYUKKE. - NO ONE YES MAN (captin1) [PEACE]](https://osu.ppy.sh/beatmapsets/1687287#osu/3448347)
+  5. [Cattle Decapitation - Your Disposal (Chanci) [Eradication]](https://osu.ppy.sh/beatmapsets/988109#osu/2207855)
+  6. [Nekomata Master - Byakuya Gentou (Feiri) [Expert]](https://osu.ppy.sh/beatmapsets/1704624#osu/3483161)
+  7. [Jun.A - The Refrain of the Lovely Great War (Duftende Pizza) [SOLO]](https://osu.ppy.sh/beatmapsets/725650#osu/1532151)
+- Hidden
+  1. [ITO KASHITARO - Yuudachi-no Ribbon (Yudragen) [Flash of Eternity]](https://osu.ppy.sh/beatmapsets/1383608#osu/2858218)
+  2. [Yasui Yousuke - YO-KAI Disco [MEIKAI ARRANGE VERSION] ([Keqing]) [aww shidd]](https://osu.ppy.sh/beatmapsets/1485831#osu/3046747)
+  3. [Chroma - Sudden Visitor (Fisky) [MAXIMUM]](https://osu.ppy.sh/beatmapsets/1251674#osu/2601556)
+  4. [REDALiCE - DEAD or DIE (Cyberia950) [Extra]](https://osu.ppy.sh/beatmapsets/1570534#osu/3207055)
+- Hardrock
```
Comment (0x84f) [inline wiki/Tournaments/NDC/2022/en.md:136]: Needs to be `Hard Rock` instead of `Hardrock` throughout all 3 wiki pages

**Review by 0x84f — COMMENTED** [review]
@walavouchey other than this i think this can go?

**Comment by 0x84f** [comment]
oh also i'd reduce the dimensions of the logo because right now its huge
image

File: wiki/Tournaments/NDC/2022/en.md [hunk 6]
```diff
@@ -0,0 +1,504 @@
...
+**[Download the mappack here!](https://tourney.huismetbenen.nl/10/mappools/f)**
+
+- Nomod
+  1. [AliA - joker (Kujinn) [NM1]](https://osu.ppy.sh/beatmapsets/1556595#osu/3179908)
+  2. [Plum - Dustwind (knowledgeking) [Hikuptah 200bpm]](https://osu.ppy.sh/beatmapsets/1480431#osu/3036867)
+  3. [The Fearless Flyers - Assassin (olc) [lets go]](https://osu.ppy.sh/beatmapsets/1580627#osu/3227488)
+  4. [Culprate - Beast (Vorso Remix) (Mir) [Hunted]](https://osu.ppy.sh/beatmapsets/1552826#osu/3172759)
+  5. [Imperial Circus Dead Decadence - Shinbatsu o Tadori Kyoukotsu ni Itaru (Kite) [Extreme]](https://osu.ppy.sh/beatmapsets/482382#osu/2039543)
+  6. [Polyphia - O.D. (Melwoine) [Overly Difficult]](https://osu.ppy.sh/beatmapsets/1276327#osu/2651794)
+  7. [jun with Alison - SUNKiSS DROP ~jun side~ ([Keqing]) [please donate for my gacha addiction]](https://osu.ppy.sh/beatmapsets/1498309#osu/3070802)
```
Comment (Joehuu) [inline wiki/Tournaments/NDC/2022/en.md:160]:
```suggestion
  7. [jun with Alison - SUNKiSS DROP \~jun side\~ ([Keqing]) [please donate for my gacha addiction]](https://osu.ppy.sh/beatmapsets/1498309#osu/3070802)
```

File: wiki/Tournaments/NDC/2022/en.md [hunk 7]
```diff
@@ -0,0 +1,504 @@
...
+- Hidden
+  1. [Hisui - Gashadokuro (KnightC0re) [Folklore]](https://osu.ppy.sh/beatmapsets/1699193#osu/3471944)
+  2. [brave elements - Massarani (Delis) [Insane]](https://osu.ppy.sh/beatmapsets/1008196#osu/2110050)
+  3. [sky_delta - Precursor (Ekoxier) [Ultimate Flow Style]](https://osu.ppy.sh/beatmapsets/747886#osu/1575750)
+  4. [cYsmix - Eden (Luminiscental) [Dark Rush]](https://osu.ppy.sh/beatmapsets/1273651#osu/2646589)
+- Hardrock
+  1. [AU - Infinite of nuclear fusion ~Mugen no Taiyou Shinkou~ (Luscent) [Extra Stage]](https://osu.ppy.sh/beatmapsets/1380266#osu/2852070)
```
Comment (Joehuu) [inline wiki/Tournaments/NDC/2022/en.md:167]:
```suggestion
  1. [AU - Infinite of nuclear fusion \~Mugen no Taiyou Shinkou\~ (Luscent) [Extra Stage]](https://osu.ppy.sh/beatmapsets/1380266#osu/2852070)
```

File: wiki/Tournaments/NDC/en.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# National Dutch Championship
+
+Index page for all of the National Dutch Championship series.
+
+- [National Dutch Championship 2022](2022)
+- [National Dutch Championship 2023](2023)
+- [National Dutch Championship 2024](2024)
+
+<!-- List the tournaments available in the directory chronologically -->
```
Comment (Joehuu) [inline wiki/Tournaments/NDC/en.md:9]:
```suggestion
```

File: wiki/Tournaments/NDC/2022/en.md [hunk 9]
```diff
@@ -0,0 +1,504 @@
+---
+tags: # yamllint disable rule:braces rule:osu-wiki-nested-structure
```
Comment (Joehuu) [inline wiki/Tournaments/NDC/2022/en.md:2]:
```suggestion
tags:
```

File: wiki/Tournaments/NDC/2023/en.md [hunk 10]
```diff
@@ -0,0 +1,530 @@
+---
+tags: # yamllint disable rule:braces rule:osu-wiki-nested-structure
```
Comment (Joehuu) [inline wiki/Tournaments/NDC/2023/en.md:2]:
```suggestion
tags:
```

File: wiki/Tournaments/NDC/2024/en.md [hunk 11]
```diff
@@ -0,0 +1,283 @@
+---
+tags: # yamllint disable rule:braces rule:osu-wiki-nested-structure
```
Comment (Joehuu) [inline wiki/Tournaments/NDC/2024/en.md:2]:
```suggestion
tags:
```

File: wiki/Tournaments/NDC/2022/en.md (hunk 5 above)
Comment (MrHeliX) [reply wiki/Tournaments/NDC/2022/en.md]: This is the first time in over 10 years I realize it's Hard Rock and not Hardrock

**Review by Joehuu — APPROVED** [review]
Looks good to me.

**Comment by Joehuu** [comment]
Just going to merge this, so updates can happen in new PRs (with corrections if they are missed).

**Pull request merged**